build.gradle

```diff
@@ -12,18 +12,20 @@ buildscript {
 
         // Dependencies (Alphabet sorted)
         android_legacy_support_version = '1.0.0'
-        android_support_version     = '1.1.0-alpha01'
+        android_support_version     = '1.1.0'
         android_ktx_version         = '1.0.1'
+        fragment_ktx_version        = '1.2.0'
         android_material            = '1.1.0-alpha02'
-        arch_component_version      = '2.1.0-alpha01'
+        arch_component_version      = '2.2.0-rc02'
```

Comment:
please put these work updates to the description of the PR as well 🙏 at first I thought there were only work related to hilt but apparently you also updated many component lib versions as well

Reply:
noted 👍